### ISSUE TITLE
Help appreciated (get_followed_streams)

### ISSUE THREAD

**Issue by Glog78** [twitch-rs/twitch_api#316]
in the scope of:

``` rust
#[tokio::main]
pub async fn main () {
// ....
let ch: Vec<twitch_api::helix::streams::Stream> = twitch.helix.get_followed_streams(&toke).try_collect().await.expect("something");
// ....
} 
```
it works fine 

in the scope of:

``` rust
#[tokio::main]
pub async fn main () {
let _thread = tokio::spawn( async move {
// ....
let ch: Vec<twitch_api::helix::streams::Stream> = twitch.helix.get_followed_streams(&toke).try_collect().await.expect("something");
// ....
}
} 
```
i get the following error message: https://termbin.com/1xeq

I am very sorry if the question might sound and look stupid i am just starting with rust.

**Comment by Emilgardis** [comment]
Hi! Thanks for the report!

I know what this error is and I'll get it fixed, the problem is that some of the functions are using a wrong bound. I'll get this fixed asap

**Comment by Emilgardis** [comment]
please try the newly released `0.7.0-rc.4` and see if that works (I'm pretty sure it does) :D 

good luck learning rust!
